Resources/Prototypes/Datasets/tips.yml

```
@@ -1,61 +1,61 @@
+  - "Здравый смысл помогает избегать конфликтов."
+  - "Другие игроки — тоже люди."
+  - "Периодически проверяйте Сингулярность, пока она запущена. Её саботаж может поставить под угрозу всю станцию."
+  - "Химикаты не реагируют между собой, пока находятся в буфере ХимМастера."
+  - "Зажмите ПРОБЕЛ (по умолчанию) чтобы замедлить движение шаттла во время пилотирования. Это позволит выполнять более точные маневры или даже полностью остановиться."
+  - "Дексалин, Дексалин плюс, и Эпинефрин будут выводить Лексорин из вашей кровеносной системы в процессе метаболизма."
+  - "Каждый член экипажа имеет в своем аварийном запасе экстренный медипен, содержищай эпинефрин и транексамовую кислоту."
+  - "ДАМ является целью с высоким приоритетом и легко саботируется. Убедитесь, что вы настроили Сингулярность или солнечные панели чтобы не остаться без энергии, если он взорвется."
+  - "С помощью ручного этикетировщика можно маркировать любые предметы, в том числе продукты питания или баночки с таблетками."
+  - "Противоударная броня значительно эффективнее обычной если противник не использует огнестрельное оружие."
+  - "Как сотрудник службы безопасности, помните, что После не значит Вследствие. Возможно кто-то просто оказался не в том месте и не в то время!"
+  - "Будучи призраком, вы можете использовать меню взаимодействия, чтобы автоматически следовать и крутиться вокруг любого объекта в игре."
+  - "Будучи предателем, вам может быть поручено устранить другого предателя, а может быть и наоборот."
+  - "Будучи предателем, вы можете использовать гарнитуру синдиката для поддержания связи с любым другим, купившим гарнитуру, предателем."
+  - "Если вы предатель, то пингвины-гренадёры не будут вас атаковать, и будут взрываться при смерти."
+  - "Будучи ядерным оперативником, держитесь в группе! Пусть ваша броня и крепка, ваши соратники гораздо лучше справятся с вашим спасением: и огнём прикроют, и в укрытие лежащего тебя оттащат."
+  - "Будучи ядерным оперативником помните, что связь имеет ключевое значение! Используйте свою рацию, чтобы поддерживать связь с вашими соратниками и координировать действия."
+  - "Вы можете осмотреть свою гарнитуру, чтобы узнать, какие радиоканалы вам доступны и как по ним говорить."
+  - "Будучи утилизатором всегда носите с собой GPS и записывайте координаты станции на случай, если ваш обломок затеряется в космосе."
+  - "Будучи барменом помните, что вы можете воспользоваться циркулярной пилой на своем ружье, чтобы его стало легче хранить."
+  - "Будучи клоуном, разнообразьте свои приколы! Никто не любит шутников с одной шуткой."
+  - "Будучи клоуном помните, что даже если вы потеряли свою банановую кожурку, то на вашем КПК всё ещё можно поскользнуться! Хонк!"
+  - "Будучи мимом помните, что ваш обет молчания - источник вашей силы. Нарушив его, вы лишитесь своих сил и чести."
+  - "Будучи адвокатом, если наказание кажется вам слишком суровым, попытайтесь обсудить это со смотрителем."
```

Нужно, что-то более нейтральное из-за отсутствия такового.
```suggestion
  - "Занимаясь судебным делом, если наказание кажется вам слишком суровым, попытайтесь обсудить это со смотрителем."
```